### PR TITLE
Document list subcommand

### LINKED ISSUE #191: Add `list` command documentation in README

### PR DESCRIPTION
**Pull request by ValentinTrinque**
Close #191

### CODE REVIEW

File: cmd/README.md [hunk 1]
```diff
@@ -74,6 +74,14 @@ specify the passphrase with the ``--passphrase`` flag.
 This command is only able to import the wallet from which you can re-generate
 your key pairs.
 
+## List registered wallet
```
Comment (candida-d) [inline cmd/README.md:77]:
```suggestion
## List registered wallets
```

File: cmd/README.md [hunk 2]
```diff
@@ -74,6 +74,14 @@ specify the passphrase with the ``--passphrase`` flag.
...
+
+If you want to list all the registered wallets, use the following command:
```
Comment (candida-d) [inline cmd/README.md:79]: Does this mean all the keypairs registered with this wallet? Or does it mean something else?

Comment (ValentinTrinque) [reply cmd/README.md:79]: That just mean the wallets. Not their keys. For the keys, use `key list`

Comment (candida-d) [reply cmd/README.md:79]: So this is for wallets within the wallet?

Comment (ValentinTrinque) [reply cmd/README.md:79]: The go-wallet project is able to manage several wallets. And each wallet has one to several keys.

**Review by jeremyletang — APPROVED** [review]

**Pull request merged**